.travis.yml

```diff
@@ -45,25 +47,23 @@ matrix:
     - php: '7.1'
       env: SYMFONY=3.3.*
     - php: '7.1'
-      env: SYMFONY=dev-master@dev
+      env: SYMFONY='dev-master as 3.3'
```

Comment:
dev-master of symfony is 4.1 at this point, i think. we should however test with 3.4 and 4.0 eventually... but lets not do too many things in the same PR.

Reply:
We should test with 3.4 directly instead of aliasing. Doing so we see the issues we have to fix and issues can also be a depending package that still does not like 3.4/4.0